flow/packet-headers/gre.yaml

```diff
@@ -42,7 +42,7 @@ components:
             format: integer
             length: 16
             default: 2048
```

Comment:
Wondering if we change the default to auto. Assuming in a test today it is not set only if the below protocol matches type 2048 which will continue to work with new default but helps avoid user having to find the value for other protocol types.

Reply:
Final generated yaml has auto by default.